src/tests/testACLMaxUserIP.cc

```diff
@@ -19,13 +19,13 @@
 
 #include <stdexcept>
 
-CPPUNIT_TEST_SUITE_REGISTRATION( testACLMaxUserIP );
+CPPUNIT_TEST_SUITE_REGISTRATION( TestACLMaxUserIP );
```

Comment:
BTW, this and three more test classes still do not obey CamelCase rules, but I did not fix them (beyond first letter capitalization) because these violations are based on the (also wrong) names of the classes this code is testing (e.g., ACLMaxUserIP). If those tested classes are renamed, we will be able to rename these testing classes as well.

```
class TestACLMaxUserIP: public CPPUNIT_NS::TestFixture
class TestDiskIO: public CPPUNIT_NS::TestFixture
class TestSBuf: public CPPUNIT_NS::TestFixture
class TestSBufList: public CPPUNIT_NS::TestFixture
```

Reply:
ACL is an acronym, as is SBuf. I wouldn't obsess over these

Reply:
> ACL is an acronym, as is SBuf. I wouldn't obsess over these

We do CamelCase most acronyms in modern Squid sources (including "ACL" and excluding "SBuf"). I would not call a desire to explain why a few non-CamelCase test class names were not converted to CamelCase (in a PR that claims to fix _all_ test class names) an obsession, but I suspect you did not mean for your comment to imply that somebody is "obsessed over these" issues.

Reply:
> > ACL is an acronym, as is SBuf. I wouldn't obsess over these
> 
> We do CamelCase most acronyms in modern Squid sources (including "ACL" and excluding "SBuf"). I would not call a desire to explain why a few non-CamelCase test class names were not converted to CamelCase (in a PR that claims to fix _all_ test class names) an obsession, but I suspect you did not mean for your comment to imply that somebody is "obsessed over these" issues.

Of course I am only referring to my own plans (and lack thereof) about these classes.